_posts/2019-03-18-speed-up-your-pipeline-using-cache.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  Speed up your pipeline using the cache
+date: 2019-03-18
+tags: codebuild aws docker cache continuous-integration
+author: JK Gunnink
+---
+
+Fast feedback cycles in development is an important consideration when deploying software often as
+it helps reduce context switching. When a developer creates some functionality, and wants to test
+and validate the functionality using a CI tool, fast feedback is important to maintain focus.
+
+In a recent project I worked on, one of the tasks I was given was to speed up the time it took for
+the CI tools to report success or failure in order to validate the work, but also get the work
```

Comment:
tool or *tools*?

Reply:
I said tools because there were two types of CI checks required for that project